.github/actions/upload/action.yml

```diff
@@ -1,10 +1,14 @@
 name: upload
-description: "Upload test results"
+description: "Upload test results and heap dump"
 inputs:
   name-suffix:
     default: ${{ github.job }}
   test-report-retention-days:
     default: 5
+  heap-dump-retention-days:
+    default: 14
```

Comment:
if we run out of space, how will we know about that?
will we be able to manually delete some heap dumps?

do we know how many heap dumps we can upload? i think current Xmx for tests implies heap dumps are 3.5 GB files

Reply:
We now upload every Trino tarball built for running product tests, which is over 1GB. The docs don't say anything about limits in public repositories: https://docs.github.com/en/actions/using-workflows/storing-workflow-data-as-artifacts
Since I'm not the repo owner, I can't check if there's some summary of usage for storage in the repo settings.

Reply:
I don't see anything like that on https://github.com/trinodb/trino/settings.

Reply:
@nineinchnick and myself checked the repo-level and org-level settings. It *seems* there is no disk quota limit being enforced right now.